portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -34,15 +37,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     long timestamp = System.currentTimeMillis();
     String userEmail = userService.getCurrentUser().getEmail();
+    float score = sentimentAnalysisScore(body);
```

Comment:
As discussed offline, it would make sense to have a separate servlet for sentiment analysis. No need to do anything in this PR, but it would be a good follow-up :)